classes/02class/exercises/c02-network08/carineoliveira/c02-network08.txt

```diff
@@ -0,0 +1,30 @@
+Class 02 - Exercise Network 08
+
+Look for the network ACL created for your VPC and answer the following:
+Your ACL rule inbound and outbound should look like this:
+
+Rule#	Type	        Protocol	Port Range	Source	        Allow/Deny
+100	    ALL Traffic	    ALL	        ALL	        <vpc-network>	ALLOW
+101	    ALL Traffic	    ALL	        ALL	        0.0.0.0/0	    ALLOW
+*	    ALL Traffic	    ALL	        ALL	        0.0.0.0/0	    DENY
+
+- What is the difference between inbound x outbound?
+    Inbound: allows traffic from a single IP address to access the instance.
```

Comment:
inbound is to control connections coming into your subnet. It can be used to allow/deny a single IP address or a network range

Reply:
thanks for the details. ;)